index.js

```diff
@@ -1,11 +1,11 @@
-import data from './data.js';
-
+import data from './modules/book.js';
+import { DateTime } from './node_modules/luxon/build/es6/luxon.js';
```

Comment:
- [ ] This line of code is stopping the synchronous flow of your logic in the `index.js` file. Kindly remove it. The project requirement states that you should use the Luxon package as a `javascript` file and then import the `DateTime` object. 

_**Hint:**_ When you add `Luxon` as a minified or full `.js` file, you can import it using:

`import { DateTime } from './luxonfilename'`